native/opencv-wrapper.h

```diff
@@ -52,25 +54,11 @@ typedef struct {
     float angle;
 } RotatedRect;
 
-typedef struct {
-    Rect* array;
-    size_t size;
-} VecRect;
-
-typedef struct {
-    double* array;
-    size_t size;
-} VecDouble;
-
-typedef struct {
-    Point2i* array;
-    size_t size;
-} VecPoint;
-
-typedef struct {
-    VecPoint* array;
-    size_t size;
-} VecPoints;
+VecType(void, Vec)
```

Comment:
Hmm. It makes sense. However, we could either remove `;` from macro definition or use it in both places and probably get compiler warning about empty staement, because it becomes `VecDoubles;;`. So I'd like to leave it as it is.

Reply:
Right, that would be an extra void statement. I think I am in favor of NOT including the semicolon in the macros definition (although, not a strong preference). This is your call, whether to change or not, I will merge the MR after I hear back from you.

Reply:
@nebgnahz ok, I have inspected some sources from public companies, such as MS, Apple and Google, and didn't find any macro that is used this way, for example: https://github.com/dotnet/coreclr/blob/master/src/vm/dllimportcallback.cpp#L74-L77 

So I changed sources. I also remind about squashing these guys :)